bandit/core/cwemap.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding:utf-8 -*-
```

Comment:
Line can be removed as UTF-8 is only for when unicode characters are in the file.